firestore/integration_test_internal/src/firestore_integration_test.cc

```diff
@@ -306,6 +303,20 @@ bool FirestoreIntegrationTest::FailIfUnsuccessful(const char* operation,
   }
 }
 
+std::vector<MapFieldValue> FirestoreIntegrationTest::QuerySnapshotToValues(
+    const QuerySnapshot& snapshot) const {
+  std::vector<MapFieldValue> result;
+  for (const DocumentSnapshot& doc : snapshot.documents()) {
+    result.push_back(doc.GetData());
+  }
+  return result;
+}
+
+class TemporaryLintErrorClass {
+  // The below line should give a lint error.
+  TemporaryLintErrorClass(int t) {}
```

Comment:
⚠️ Lint warning: `Single-parameter constructors should be marked explicit.`<hidden value="cpplint-file-comment"></hidden><hidden value="firestore/integration_test_internal/src/firestore_integration_test.cc:317:  Single-parameter constructors should be marked explicit.  [runtime/explicit] [5]"></hidden>